Source/Controllers/MainViewControllers/TableStructure/SPTableStructure.m

```diff
@@ -1183,6 +1207,11 @@ - (NSDictionary *)tableSourceForPrinting
 	return [NSDictionary dictionaryWithObjectsAndKeys:tempResult, @"structure", tempResult2, @"indexes", nil];
 }
 
+- (NSMutableArray *)activeFieldsSource
+{
+	return filteredTableFields == nil ? tableFields : filteredTableFields;
```

Comment:
Can happen that filteredTableFields won't be nil but will be empty? If so, should we also check empty?

Reply:
Yes empty can happen. This would mean that the user got no matches in their filter -- user can remove filter to get all results back. From a filtering perspective, I think this provides better feedback to the user i.e. filtering for something that is not in the field names gives you an empty table rather than all the fields back.

Reply:
That makes sense. Empty should be fine, playing with it locally that behavior you described @luis- felt correct!